cmd/kgctl/showconf.go

```diff
@@ -151,14 +152,14 @@ func runShowConfNode(_ *cobra.Command, args []string) error {
 		}
 	}
 
-	t, err := mesh.NewTopology(nodes, peers, opts.granularity, hostname, opts.port, []byte{}, subnet, nodes[hostname].PersistentKeepalive, nil)
+	t, err := mesh.NewTopology(nodes, peers, opts.granularity, hostname, int(opts.port), wgtypes.Key{}, subnet, nodes[hostname].PersistentKeepalive, nil)
```

Comment:
Do we want to directly redefine the port as an int so we don't have to convert from uint? If we want to limit values we can always add a validation function

Reply:
Maybe it makes more sense, because ports can't be higher than 2**16 and we didn't check this before.

Reply:
ok added a check for the valid port range 1-2**16-1